sdcm/mgmt.py

```diff
@@ -231,6 +231,18 @@ def status(self):
         # │ repair/dd98f6ae-bcf4-4c98-8949-573d533bb789 │                               │ 3    │            │ DONE   │
         # ╰─────────────────────────────────────────────┴───────────────────────────────┴──────┴────────────┴────────╯
 
+    @property
+    def arguments(self):
+        """
+        Gets the task's arguments
+        """
+        cmd = "task list -c {}".format(self.cluster_id)
+        res = self.sctool.run(cmd=cmd)
+        arguments_string = self.get_property(parsed_table=res, column_name='arguments')
```

Comment:
would this get us all the arguments of all tasks ?, no only the one we care about ?

Reply:
I'm not sure what you meant. if you meant wether it gives us the argumant of the task we wanted, than yes, it will. Take a look at the get_property function, it uses the task's id to find the right row in the table.
If you meant what kind of arguments we will get, it's only the optional ones, i.e. the ones we care about.